pynest/nest/__init__.py

```diff
@@ -25,6 +25,7 @@
 
 import sys
 import os
+from copy import copy
```

Comment:
For consistency with other imports, I would suggest `import copy` here and `copy.copy()` below.